Sage/carbon
# FE-2306 - Fix sticky footer flickering issue

What has been done:
- removed while loop in form.component.js - legacy loop. I think no need anymore
- add configuration to test `fixtures` in vscode debug mode
- add knob `sitckyFooter` to form stories to allow test sticky behavior
- update test

# Testing Instructions
- go to: `storybook/experimental/form/fieldset-textbox`
- set stickyFooter to be true
- resize knob window to let you see stickyFooter works
- start resizing knob window back slowly
- the sticky footer shouldn't start flickering

- Before, the sticky footer was flickering when it was close under the form
